src/app/course/team-leader-board/team-leader-board.component.scss

```diff
@@ -0,0 +1,207 @@
+// Reused elements
```

Comment:
Wondering if the css can be in a spot where it can be used by both the team and individual leaderboard as they are largely duplicated. Then can add the unique css on a component basis.